src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -145,7 +147,10 @@ const UserProfileModal: React.FunctionComponent<PropTypes> = ({
   );
 };
 
-const Badges: React.FC<{ user: WithId<User> }> = ({ user }) => {
+const Badges: React.FC<{ user: WithId<User>; currentVenue: WithId<Venue> }> = ({
```

Comment:
Would it be possible to pull this out into a separate component file for this PR? Feels 'hidden away' at the moment.

Reply:
Yep! Doing that now.